pokesummary/displaying.py

```diff
@@ -147,3 +147,22 @@ def display_summary(pokemon_name, pokemon_stats):
     type_defenses = calculate_type_defenses(pokemon_stats)
     print(get_type_defenses_chart(type_defenses))
     print()
+
+    #show pokemon pre evolution
+    while(pokemon_stats["pre_evo_id"]):
+        pre_evo = display_pre_evo(pokemon_stats["pre_evo_id"])
+        pre_evo_name = pre_evo["pokemon_name"]
+        pre_evo_details = pokemon_stats["evo_details"]
+        print(f"{pre_evo_name} evolves into {pokemon_name} at {pre_evo_details or 'trade'}")
+        pokemon_stats=pre_evo
```

Comment:
This is quite confusing; consider creating a new variable instead of reusing the parameter.